lib/deploy.js

```diff
@@ -165,30 +175,34 @@ Deploy.prototype.execute_cmds = function(cmds) {
 
 Deploy.prototype.generate_provider_file = function() {
   var result = "";
-  result = "web3.setProvider(new web3.providers.HttpProvider('http://" + this.blockchainConfig.rpcHost + ":" + this.blockchainConfig.rpcPort + "'));";
-  result += "web3.eth.defaultAccount = web3.eth.accounts[0];";
-
+  result = "var Web3 = require('web3'); var web3 = new Web3;";
+  result += "web3.setProvider(new web3.providers.HttpProvider('http://" + this.blockchainConfig.rpcHost + ":" + this.blockchainConfig.rpcPort + "'));";
```

Comment:
Either way, this seems like something for another issue...if you want to play with it in a dev environment on the test network...why not? But you can probably configure it in a better way on the blockchain.yml...perhaps something in the README? Either way...seems to me to be a separate issue. I'm just writing to what is currently available.